astropy/time/formats.py

```diff
@@ -573,6 +573,69 @@ def value(self):
 
         return iterator.operands[-1]
 
+    def to_value(self, timezone=None):
+        """
+        Convert to (potentially timezone-aware) `~datetime.datetime` object.
+
+        If ``timezone`` is not `None`, return a timezone-aware datetime
+        object.
+
+        Parameters
+        ----------
+        timezone : {`~datetime.tzinfo`, None} (optional)
+            If not `None`, return timezone-aware datetime.
+
+        Returns
+        -------
+        `~datetime.datetime`
+            If ``timezone`` is not `None`, output will be timezone-aware.
+        """
+        if timezone is not None:
+            if self._scale != 'utc':
+                raise ScaleValueError("Scale is {}, must be UTC."
```

Comment:
Also, didn't we have to write `{0}` for python 2.6 compatibility? I think the reason this was not caught by travis is that the exception is actually not covered by a test: could you add a test along the lines of
```
     with pytest.raises(ScaleValueError):
        time.tdb.to_datetime(tz)
```